hack/update/debian_version/debian_version.go

```diff
@@ -29,36 +29,41 @@ var (
 	schema = map[string]update.Item{
 		"deploy/kicbase/Dockerfile": {
 			Replace: map[string]string{
-				`UBUNTU_JAMMY_IMAGE=.*`: `UBUNTU_JAMMY_IMAGE="{{.LatestVersion}}"`,
+				`DEBIAN_BOOKWORM_IMAGE=.*`: `DEBIAN_BOOKWORM_IMAGE="{{.LatestVersion}}"`,
```

Comment:
We cannot replace the ubuntu image with debian and continue to call the command "ubuntu_version".